libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -244,10 +244,6 @@ singleton AP_Baro method get_pressure float
 singleton AP_Baro method get_temperature float
 singleton AP_Baro method get_external_temperature float
 
-include AP_ESC_Telem/AP_ESC_Telem.h
```

Comment:
Why have you removed this?

Reply:
Again, this is just to make CI pass, this is part of the last commit, that should not get merged.